integration/helpers/create-fixture.ts

```diff
@@ -316,33 +316,14 @@ export async function createFixtureProject(
   init: FixtureInit = {},
   mode?: ServerMode
 ): Promise<string> {
-  let template = "node-template";
+  let template = "vite-template";
```

Comment:
We shouldn't need this node-template anymore - deleted it and changed the default to use the vite-template